packages/hachis/hachis.20240918/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Hash sets and hash maps"
+maintainer: ["François Pottier"]
```

Comment:
```suggestion
maintainer: ["François Pottier <francois.pottier@inria.fr>"]
```
Is that all right if I add the email here? The other lint errors can be ignored

Reply:
Sure. I'll make the same change on my end.